corehq/apps/users/bulk_download.py

```diff
@@ -165,24 +165,30 @@ def count_users_and_groups(domain, user_filters, group_memoizer):
 
 
 def dump_usernames(domain, download_id, user_filters, task):
-    writer = Excel2007ExportWriter(format_as_text=True)
     users_count = get_commcare_users_by_filters(domain, user_filters, count_only=True)
     DownloadBase.set_progress(task, 0, users_count)
 
     usernames = get_mobile_usernames_by_filters(domain, user_filters)
 
-    use_transfer = settings.SHARED_DRIVE_CONF.transfer_enabled
     filename = "{}_users.xlsx".format(domain)
+    headers = [('users', [['username']])]
+    rows = [('users', [[username] for username in usernames])]
+    _dump_xlsx_and_expose_download(filename, headers, rows, download_id, task, users_count)
+
+
+def _dump_xlsx_and_expose_download(filename, headers, rows, download_id, task, total_count):
```

Comment:
No need to change, but I wonder if this function would be useful outside of this module. Every time I have to add an excel upload/download I grep around for a while and find multiple ways to do it; I feel like it's an area where wee could use more consistency.

Reply:
I think the way we align the content and headers is different? though i am not completely sure.
Well at least now there is a method so if we find something like this, we can replace with this method